backend/src/main/java/ee/tlu/forum/service/UserServiceInterface.java

```diff
@@ -6,7 +6,7 @@
 import java.util.List;
 
 public interface UserServiceInterface{
-    User editUser(User user);
+    User editUser(User user, String token);
     User registerUser(User user);
     Role saveRole(Role role);
```

Comment:
Token for this line and the next?

Reply:
Yes, as admins and users have different permissions but can access the same endpoint